.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "Resources/public/js/vendor"
```

Comment:
if we use assetic / recommend to use some node toolchain to combine files, we should not drop these into public i think. unless there are also images or other files that get accessed directly.